elastic/kibana
# [Synthetics] update edit monitor link in Synthetics App

## Summary

Resolves https://github.com/elastic/kibana/issues/142394.

![image](https://user-images.githubusercontent.com/11356435/193305450-e875751e-9fa5-423f-9d62-f8214cff0dd7.png)


### Testing
1. Create a monitor
2. Navigate to Overview page. Click on the actions popover and click edit monitor. It should take you to the Synthetics App's edit monitor flow
3. Navigate to the Synthetics Monitor Management page. Click on the actions popover and click edit monitor. It should take you to the Synthetics App's edit monitor flow
